Mathlib/Topology/MetricSpace/HolderNorm.lean

```diff
@@ -0,0 +1,225 @@
+/-
+Copyright (c) 2024 Kexing Ying. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kexing Ying
+-/
+import Mathlib.Topology.MetricSpace.Holder
+
+/-!
+# Hölder norm
+
+This file defines the Hölder (semi-)norm for Hölder functions alongside some basic properties.
```

Comment:
Please also add here a description of what that norm is. And explain the link with Hölder continuity.